MANIFEST.in

```diff
@@ -0,0 +1,8 @@
+graft src/python/grpcio/grpc
```

Comment:
That would be amazing.

On Tue, Dec 15, 2015 at 12:21 PM Masood Malekghassemi <
notifications@github.com> wrote:

> In MANIFEST.in
> <https://github.com/grpc/grpc/pull/4457#discussion_r47692695>:
>
> > @@ -0,0 +1,8 @@
> > +graft src/python/grpcio/grpc
>
> Turns out that there's a way to rename the MANIFEST.in file; if I can
> find all the configuration settings required for the tools to be aware of
> the change I'll go with that.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/grpc/grpc/pull/4457/files#r47692695>.
>